### ISSUE TITLE
attached commands broke

### ISSUE THREAD

**Issue by CloudeLecaw** [Zrips/CMI#7222]
### Description of Issue

attached commands broke
checking there NTB contains nonesen

![Snag_1127bf1](https://user-images.githubusercontent.com/12080017/190827527-8014dd06-189b-4d34-a3b1-483e43d442ab.png)
This is how the item looks when looking with /cmi attached commands

Currently All our items with attached commands are effected

### Version Information
![Snag_1145536](https://user-images.githubusercontent.com/12080017/190827681-9b74edde-9117-458b-989e-e2cfea117f34.png)

### Errors

![Snag_1151096](https://user-images.githubusercontent.com/12080017/190827740-b90c275c-3481-466f-9a3c-61c591b5866c.png)

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by mrfloris** [comment]
stop the server, find the old security.key file from a backup and replace it with the one it is using now, you might notice they are both different filesize, and start the server again.  Does that resolve this?

**Comment by Blueeyestar** [comment]
> stop the server, find the old security.key file from a backup and replace it with the one it is using now, you might notice they are both different filesize, and start the server again. Does that resolve this?

I did what you described and it worked. Thanks for the tip, the function is working again :)

**Comment by CloudeLecaw** [comment]
> stop the server, find the old security.key file from a backup and replace it with the one it is using now, you might notice they are both different filesize, and start the server again. Does that resolve this?

well it trally seems ti fix the issue...
happy thaz it works now.

well i just dont ubderstood what this does have to do with the secure.key file?  well its a strange behave if something with the key doesnt work that only attached commands. would be make morr sense if all function stop to work if the key file become invailed..

by the way.. hiw didnthis happend? the update where this issue was mentions was some time ago, and at this time we were not effected....

**Comment by Zrips** [comment]
There was no direct change relating to attached commands in recent updates. Was there any errors on server startup wen this started to happen?

**Comment by mrfloris** [comment]
> no. difinalty not.
> 
> Well More interesting would be the thing.. why this exchanged of the key file solved it... well looks like mrfloris knows the magic behind as he was giving the solution to this :P

An older version of CMI had a serious bug which got hotfixes right away. I recognized the situation and wondered if it was the same issue with the .key file. glad to hear restoring the old key file resolved it. No idea why it would have caused this. 

Which version did you upgrade from? 

edit: I just updated my live server from the .0 to .1 release, and checked the file, phew, nothing crazy for me thankfully. And ingame /cmi attachcommand thankfully shows decrypted plaintext.
